src/ucs/memory/memory_type.h

```diff
@@ -18,7 +18,6 @@ BEGIN_C_DECLS
  */
 #define UCS_MEMORY_TYPES_CPU_ACCESSIBLE \
     (UCS_BIT(UCS_MEMORY_TYPE_HOST) | \
-     UCS_BIT(UCS_MEMORY_TYPE_CUDA_MANAGED) | \
```

Comment:
is it some unrelated bugfix?

Reply:
It's related. We need this change to let cuda_copy be used for staging. Otherwise memcpy is used to copy from managed to bounce buffer.